projects/packages/woocommerce-analytics/src/class-universal.php

```diff
@@ -76,27 +73,6 @@ public function init_hooks() {
 		add_action( 'wp_footer', array( $this, 'capture_page_view' ), 11 );
```

Comment:
If initial events are [not tracked](https://github.com/Automattic/jetpack/blob/3522253ead4928f3b9c9ff6e7eaa19d6309ac113/projects/packages/woocommerce-analytics/src/class-woo-analytics-trait.php#L354), is it still necessary to hook on the action here? Or Am I missing something?